forge-gui/res/cardsfolder/upcoming/no_one_left_behind.txt

```diff
@@ -1,9 +1,9 @@
 Name:No One Left Behind
 ManaCost:4 B
 Types:Sorcery
-S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ CostReduction | Relative$ True | EffectZone$ All | Description$ This spell costs {3} less to cast if it targets a blue permanent you don't control.
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ CostReduction | Relative$ True | EffectZone$ All | Description$ This spell costs {3} less to cast if it targets a creature card with mana value 3 or less.
 SVar:CostReduction:Count$Compare CheckTgt GE1.3.0
-SVar:CheckTgt:TargetedObjects$Valid Creature.cmcLE3+YouOwn
+SVar:CheckTgt:TargetedObjects$Valid Creature.cmcLE3+YouCtrl
```

Comment:
Since the SP$ says your graveyard, the YouOwn is fine there... for this static though it just cares about `creature card with mana value 3 or less`

Reply:
Oh I see it's just a check for creatures in your yard